### PR TITLE
Add correct config info to applicationData

### PR DESCRIPTION
**Pull request by CarlosNZ**
Quick fix for the missing logo issue in the Company Data View.

The applicationData that was being passed to the "SummaryView" components in Data Views was incorrect due to a recent change to how the server urls are generated. I'd updated all the other applicationData's, but somehow missed this one.

Have also updated the comment to make it clearer what's going on here.

### CODE REVIEW

**Review by nmadruga — CHANGES_REQUESTED** [review]
I'm not sure if it's associated but the file that has been previously uploaded in the example "Create organisation" serial ORG-0046-2022 on the latest snapshot doesn't open correctly.
I can see that the file is present in the data base with unique_id `3dLFqxr27Q2a52F3IMoFc`, but this is displayed when I hover the link to this document (during Review):
`http://localhost:8080/api/public/file?uid=undefined` 

Well, I just realised that this is the same URL showing for any of the images/documents in the form.

**Comment by CarlosNZ** [comment]
> I'm not sure if it's associated but the file that has been previously uploaded in the example "Create organisation" serial ORG-0046-2022 on the latest snapshot doesn't open correctly. I can see that the file is present in the data base with unique_id `3dLFqxr27Q2a52F3IMoFc`, but this is displayed when I hover the link to this document (during Review): `http://localhost:8080/api/public/file?uid=undefined`
> 
> Well, I just realised that this is the same URL showing for any of the images/documents in the form.

Okay, good catch -- but not related to this issue (it's on develop like that). I'll investigate.

**Comment by CarlosNZ** [comment]
> > I'm not sure if it's associated but the file that has been previously uploaded in the example "Create organisation" serial ORG-0046-2022 on the latest snapshot doesn't open correctly. I can see that the file is present in the data base with unique_id `3dLFqxr27Q2a52F3IMoFc`, but this is displayed when I hover the link to this document (during Review): `http://localhost:8080/api/public/file?uid=undefined`
> > Well, I just realised that this is the same URL showing for any of the images/documents in the form.
> 
> Okay, good catch -- but not related to this issue (it's on develop like that). I'll investigate.

It's a quick fix, fortunately, so I've just squeezed it in here: 49dd92bf

It's annoying though, I thought I had the typing really strict in the "getServerUrl" method, so I didn't think incorrect parameters like this could have slipped through. Hmmmm....

File: src/formElementPlugins/fileUpload/src/SummaryView.tsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ import getServerUrl from '../../../utils/helpers/endpoints/endpointUrlBuilder'
 import { SummaryViewProps } from '../../types'
 
 const SummaryView: React.FC<SummaryViewProps> = ({ parameters, Markdown, response }) => {
+  console.log('files', response?.files)
```
Comment (nmadruga) [inline src/formElementPlugins/fileUpload/src/SummaryView.tsx]: Needs removing!

**Review by nmadruga — COMMENTED** [review]
I wasn't able to properly test... But let's get it in for the server upgrade

**Review by nmadruga — APPROVED** [review]

**Pull request merged**